### DISCUSSION TITLE
What do you do with (probably) incorrect QSOs?

### DISCUSSION THREAD

**Discussion by Werzi2001** [magicbug/Cloudlog#1233] [Q&A]
I have some QSOs which are probably incorrect (e.g. name on QRZ does not match the name the operator had or the distance to the grid locator is too big). Just wondering what you do with such QSOs? Just delete them? But what if a QSL (e.g. on eQSL) appears that "solves" the error (e.g. a typo in the call sign)? On one hand I don't want to have QSOs in the log which are most definitely incorrect but on the other hand I don't want to loose the information that there was "something".

**Comment by dg9vh** [comment]
Hi,

If you are sure that the QSO is not correct and valid, you can use the
notes tab to fill in there some internal information about this. So you
have the way to get it staying in the log but have a note about the
situation.

73 de Kim
DG9VH

Werzi2001 ***@***.***> schrieb am Sa., 16. Okt. 2021, 23:22:

**Comment by m0pcb** [comment]
I always keep the QSO, quite often the name given on the air won’t match QRZ.com. More than a few times I’ve received a QSL card and been able to correct a callsign in my logbook, or even the QSO date, and that’s been years after the QSO.

If in doubt, log it. 😊

73, Iain M0PCB / AD5XI

**Comment by Werzi2001** [comment]
Thanks a lot for your input @dg9vh and @m0pcb . Based on that I decided to leave them in the log and use the note feature to write some details about why I think they are incorrect. In addition to that I'll use a "incorrect QSOs" station profile to group them. Using that the (most definitely) incorrect QSOs won't mess up my statistics. For example I have an incorrect QSO which sets the "distance worked record" to 600km on 2m which is not right ;)

**Comment by AndreasK79** [comment]
I would also leave them in there, and set DXCC to none and CQ zone to 0. At least they will not be counted in the awards.

**Comment by AndreasK79** [comment]
Perhaps a flag in the database for marking QSOs as invalid, would be the way to go.

**Comment by dg9vh** [comment]
This flag is the best way I think :-) If you find a suitable place to put
it in :-) do it.

Am So., 17. Okt. 2021 um 11:37 Uhr schrieb Andreas Kristiansen <
***@***.***>:

> Perhaps a flag in the database for marking QSOs as invalid, would be the
> way to go.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/discussions/1233#discussioncomment-1488779>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKN5OWC4IB7ULWYYCRVN73UHKIV7ANCNFSM5GEANRUQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
Ortsverband Q03, Neunkirchen

**Comment by AndreasK79** [comment]
@dg9vh it probably is. Just need to exclude the invalid QSOs in every query that is relevant... @magicbug is this something we should add?
